### DISCUSSION TITLE
TrueNas Scale - Location of Jellyfin Config folders and how to get there..??

### DISCUSSION THREAD

**Discussion by Zacapa25** [jellyfin/jellyfin#8830] [Q&A]
Hi Guys,

Im running TruNas Scale on my server, and after I was updating to Truenas Scale 22.12-RC.1 I have to login to Jellyfin again, 
but it seems that Jellyfin will not accept my password to login  (or maybe I'm not remembering correct).

I requsted for a new password and this is sent to my server with instructions on how to reset, the problem now is that I can't find
where the Jellyfin cofig folder is located on Scale, and also I dont know how to access it.

Maybe someone in here has similar issues, or knows something about it.

Cheers

**Comment by pypp** [comment]
Never used TrueNAS, but I'm guessing the config is located at /var/cache/jellyfin.

**Reply by Zacapa25** [reply]
Hi 1netanel,

Thanks for your answer,
Yes you're right, 
but on TrueNas Scale this is different and these folders are hidden, so its not that easy compare to synology, windows etc.
Scale is Linux and I'm not that experienced with that so far.

But thanks anyway.

**Reply by lenn219** [reply]
how is your storage setup in the option for the service.

**Reply by Zacapa25** [reply]
Hi lenn219,
what exactly do you mean by that...??

I have a pool, on this there is datasets and the ix-applications data set.
these ix data needs, as far as I can read in different forums, an additional app (ex: True command) to gain access.

A second problem I have right now is that I don't have any applications under "available applications" so this means that I'm not able to
install further apps, and of course then also not "True command".
BTW : this issue is later confirmed as a bug.

I think that I will wait to do anything further until the real Release of this Scale version will be released, which is on December 13-2022.
Hopefully this will then solve my issues.

Cheers
